ebos/eclgenericoutputblackoilmodule.cc

```diff
@@ -943,14 +997,19 @@ doAllocBuffers(unsigned bufferSize,
         }
     }
 
+    if (auto& norst = rstKeywords["NORST"]; norst > 0) {
+        // Don't emit diagnostic messages about unsupported 'NORST' key.
+        norst = 0;
+    }
```

Comment:
This seem new and not whitespace. Intentional?

Reply:
> This seems new and not whitespace. Intentional?

It's intentional as I want to remove the misleading diagnostic
```
Keyword NORST is unsupported for output to file
```
but you're absolutely right that this is new code and not whitespace related.  I can split this off to a separate PR if you'd like.